src/openrct2-ui/interface/Theme.cpp

```diff
@@ -137,6 +137,7 @@ static constexpr WindowThemeDesc WindowThemeDescriptors[] =
     { WindowClass::SavePrompt,                  "WC_SAVE_PROMPT",                    STR_THEMES_WINDOW_SAVE_PROMPT,                    COLOURS_1(translucent(COLOUR_BORDEAUX_RED)                                                                                                             ) },
     { WindowClass::ConstructRide,               "WC_CONSTRUCT_RIDE",                 STR_THEMES_WINDOW_CONSTRUCT_RIDE,                 COLOURS_3(opaque(COLOUR_DARK_BROWN),               opaque(COLOUR_BORDEAUX_RED),             opaque(COLOUR_BORDEAUX_RED)                                ) },
     { WindowClass::DemolishRidePrompt,          "WC_DEMOLISH_RIDE_PROMPT",           STR_THEMES_WINDOW_DEMOLISH_RIDE_PROMPT,           COLOURS_1(translucent(COLOUR_BORDEAUX_RED)                                                                                                             ) },
+    { WindowClass::Error,                       "WC_ERROR",                          STR_THEMES_WINDOW_ERROR,                          COLOURS_1(translucent(COLOUR_BORDEAUX_RED)                                                                                                             ) },
```

Comment:
Bordeaux Red (same as demolish prompt):
![Diamond Heights 2024-07-25 16-48-20](https://github.com/user-attachments/assets/5fb22676-2781-4626-a66b-c3978c9c4769)

Bright Red (looks a bit more error-y?):
<img width="396" alt="Screenshot 2024-07-25 at 18 38 12" src="https://github.com/user-attachments/assets/8ea47b07-b10b-4ff4-843e-6906c13a0aab">

Reply:
I would prefer one that meets [accessibility guidelines](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html):

> The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, [...]

| Scenario | Contrast | 
|--|--|
| Pink text with bordeaux red on grass | [5.4](https://webaim.org/resources/contrastchecker/?fcolor=FFBFBF&bcolor=8F2727) |
| Pink text with bright red on grass | [4.76:1](https://webaim.org/resources/contrastchecker/?fcolor=FFDBD7&bcolor=C70200) |
| Pink text with bordeaux red on sand | [3.24:1](https://webaim.org/resources/contrastchecker/?fcolor=FFBFBF&bcolor=B34F4F) |
| Pick text with bright red on sand | ? | 

Some more tests would be nice but I'm not available in the coming days to test it ingame myself unfortunately. A test with snow would be interesting too. 😊

Reply:
Not sure if that’s really necessary, as people can now change it themselves. If people have trouble reading, I expect them to disable the transparent background to begin with.